app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -49,17 +49,41 @@ def store_saml_request
     ).call
   end
 
-  def requested_authn_context
+  def requested_authn_contexts
     @requested_authn_context ||= saml_request.requested_authn_contexts.presence ||
                                  [default_authn_context]
   end
 
+  def requested_authn_context
+    if AppConfig.env.aal_authn_context_enabled == 'true'
+      requested_aal_authn_context
+    else
+      requested_ial_authn_context
+    end
+  end
+
   def default_authn_context
+    if AppConfig.env.aal_authn_context_enabled == 'true'
+      default_aal_context
+    else
+      default_ial_context
+    end
+  end
+
+  def default_aal_context
+    Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF
```

Comment:
Maybe a better discussion for Slack, but should we default to AAL1?

Reply:
Do we support AAL1? I added the constants, but I thought we defaulted at AAL2...